src/components/ProjectView/ImageCarousel.vue

```diff
@@ -53,7 +57,7 @@ const handleThumbnailClick = (item, index) => {
     >
       <div class="carousel__item">
         <img
-          :src="item.url"
+          :src="largeImageSrc ? largeImageSrc : store.state.firstLargeImage"
```

Comment:
related to my previous comment about the getter but if we have a getter then this can just be `:src="store.getters.getLargeImageSrc(smallImageBasename)"` and we don't need the ref or function `getLargeImageSource()`